.github/workflows/tests-e2e-npm.yml

```diff
@@ -75,8 +79,9 @@ jobs:
         if: always()
         with:
           name: playwright-report
-          path: ${{ runner.temp }}/playwright-report/
+          path: ./playwright-report/
```

Comment:
surely we're setting `VERITABLE_E2E_OUT_DIR` to include the `runner.temp` so we should here still?

Reply:
so thing is I am not sure if we still need it - because it was only used inside a `docker-compose-e2e.yml` to put files from within a container to a spec. location but we are not doing this anymore. 
I think this confirms it: 
- [this is a run](https://github.com/digicatapult/veritable-ui/pull/235/commits/4bbc3e378dcc7ccd8954dfba3483d00e87bbdb96) I did with the testcontainers  which uploaded the artifact [if you look at this](https://github.com/digicatapult/veritable-ui/actions/runs/11822228135/job/32938794038)

Reply:
possibly true but then we shouldn't set `VERITABLE_E2E_OUT_DIR` right?